skills/apple/swiftui-whats-new-27/references/document-based-apps.md

```
@@ -0,0 +1,530 @@
+# Document-Based Apps: `ReadableDocument` / `WritableDocument`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)